src/cljs/harja/views/hallinta/kojelauta.cljs

```diff
@@ -171,7 +184,12 @@
      :muokattava? (constantly false)
      :nimi :ks_tila :leveys 15
      :tyyppi :komponentti
-     :komponentti (fn [rivi] [valikatselmus-sarake rivi])}]
+     :komponentti (fn [rivi] [valikatselmus-sarake rivi])}
+    {:otsikko "Lupausten tavoite\u00ADpiste\u00ADmäärä"
```

Comment:
<details>
<summary> ᴄʟɪᴄᴋ ᴛᴏ ᴇxᴘᴀɴᴅ </summary>

Gridille taas ehdotuksia, jos urakan leveys 8, ja hoitovuosi leveys 7, saisi taas pienemmällekkin resolla  hyvän näköinen, toimii myös full sized.

Tältä näyttäis, jos olis tuo lyhyt_nimi myös mukana (katsoin että ei minusta ole salaista tietoa joten uskallan laittaa screenshotin):

![image](https://github.com/user-attachments/assets/bcb09287-8bb3-4c67-83b2-0580391c7633)
</details>